idd/Energy+.idd.in

```diff
@@ -16508,57 +16519,50 @@ SurfaceProperty:ExposedFoundationPerimeter,
        \minimum 0.0
        \maximum 1.0
        \default 1.0
```

Comment:
Never mind - github scammed me - not sure why - it skipped some lines with an expand button in the middle, so I didn't realize N2 was even there.  Now I see it in the full file.